### PR TITLE
Code style standardization according to linter-flake8

### PR DESCRIPTION
**Pull request by jacekgrzybek**
Code corrected with guidelines from flake8

Description

Types of changes:
-spreading the lines so that they do not exceed 80 characters
-add new whitespaces
-add new lines
-remove unused components

Checklist:
- [x] correct except in cmd_helper.py
- [x] check in max_bg_threads_env_var_test.py in message is undefined variable thread_count

---
This change is [Reviewable](https://reviewable.io/reviews/memkind/memkind/611)

### CODE REVIEW

**Review by bratpiorka — COMMENTED** [review]
> Reviewed 10 of 10 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/611)* status: all files reviewed, 1 unresolved discussion (waiting on @jacekgrzybek)

___
*[test/autohbw_test.py, line 2 at r1](https://reviewable.io/reviews/memkind/memkind/611#-MX6FoknASYs5Zvqem4G:-MX6Foko8vQ_SButhm64:b-rmocyw) ([raw file](https://github.com/memkind/memkind/blob/5aeadc905027220ecac6f15cc2ae716592c25957/test/autohbw_test.py#L2)):*
> ```Python
> # SPDX-License-Identifier: BSD-2-Clause
> # Copyright (C) 2016 - 2020 Intel Corporation.
> ```

bump date here and in other files

File: test/max_bg_threads_env_var_test.py [hunk 1]
```diff
@@ -16,35 +17,47 @@ def min_threads(self):
...
            correctly."""
         self.threads_limit = -1
         output, retcode = self.run_test_binary()
         assert retcode == 134, \
-               self.fail_msg.format(f"Error: Negative value of MEMKIND_BACKGROUND_THREAD_LIMIT should not be handled. " \
-                                    f"Counted threads: {threads_count}. Output: {output}")
+            self.fail_msg.format(
+                f"Error: Negative value of MEMKIND_BACKGROUND_THREAD_LIMIT",
+                " should not be handled. ",
+                f"Counted threads: {threads_count}. Output: {output}")
```
Comment (PatKamin) [inline test/max_bg_threads_env_var_test.py:63]: These changes should be a part of the previous commit. A separate commit should include a fix for this message.

**Review by bratpiorka — COMMENTED** [review]
> Reviewed 7 of 7 files at r3, 1 of 1 files at r4, 1 of 1 files at r5.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/611)* status: all files reviewed, 7 unresolved discussions (waiting on @jacekgrzybek)

**Review by bratpiorka — COMMENTED** [review]
> Reviewed 1 of 1 files at r6.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/611)* status: all files reviewed, 7 unresolved discussions (waiting on @jacekgrzybek)

**Review by michalbiesek — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/611)* status: 8 of 13 files reviewed, 7 unresolved discussions (waiting on @bratpiorka and @jacekgrzybek)

___
*[test/autohbw_test.py, line 7 at r3](https://reviewable.io/reviews/memkind/memkind/611#-MY4pxr5-FcvmacPP3i9:-MY4pxr5-FcvmacPP3iA:b-rkcmhx) ([raw file](https://github.com/memkind/memkind/blob/d43701ec5506967d6413acfa469d3f895ddb3221/test/autohbw_test.py#L7)):*
> ```Python
> from python_framework import CMD_helper
> ```

Please update your commits (author part):
use the same email account in git config as for Github Account - use intel one, please.

**Review by bratpiorka — COMMENTED** [review]
:lgtm:

> Reviewed 5 of 5 files at r8, 4 of 4 files at r9, 1 of 1 files at r10.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/611#-:-MYERMwA2M7wYrDMBavE:bnfp4nl)* status: all files reviewed, 7 unresolved discussions (waiting on @jacekgrzybek)

**Review by bratpiorka — COMMENTED** [review]
> Reviewed 1 of 1 files at r11, 8 of 9 files at r12, 2 of 2 files at r13, 3 of 3 files at r14.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/611)* status: all files reviewed, 7 unresolved discussions (waiting on @jacekgrzybek)

**Review by michalbiesek — CHANGES_REQUESTED** [review]
> Reviewed 3 of 9 files at r12.
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/611)* status: all files reviewed, 8 unresolved discussions (waiting on @jacekgrzybek)

___
*[test/autohbw_test.py, line 7 at r3](https://reviewable.io/reviews/memkind/memkind/611#-MZSdZTh3OihnY2LvWZi:-MZSdZTh3OihnY2LvWZj:b68l1d6) ([raw file](https://github.com/memkind/memkind/blob/d43701ec5506967d6413acfa469d3f895ddb3221/test/autohbw_test.py#L7)):*
LGTM, but do as @patkamin suggested, in the last commit("Add python linter to GitHub Actions") - modify the Github Actions file. Changes that are performed in the other files should be included in the commit ("Increasing the readability of the code according to the linter-flake")
The one way would be:
<details><summary><i alt="summary">Quoted 6 lines of code…</i></summary>

> ```
> step 0 **for your consideration, you could backup your work in a separate branch**
> step 1 call git reset --soft HEAD~1"
> step 2 perform the first commit with changes in gha.yml
> step 3 perform the second commit with changes in the rest of the files (python files)
> step 4 call git rebase --interactive HEAD~4
> step 5 move the commit you added in step 3 after "Increasing the readability of the code according to the linter-flake") and change > option from pick to fixup
> ```
</details>


See https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History for details regarding step 5
___
*[test/autohbw_test.py, line 8 at r3](https://reviewable.io/reviews/memkind/memkind/611#-MZSiaYEBiJnJ-SS9i16:-MZSiaYEBiJnJ-SS9i17:bd9xec1) ([raw file](https://github.com/memkind/memkind/blob/d43701ec5506967d6413acfa469d3f895ddb3221/test/autohbw_test.py#L8)):*
> ```
> def _get_libautohbw_path():
>     for p in ["/usr/lib64/libautohbw.so.0", "/usr/lib/libautohbw.so.0"]:
>         if os.path.isfile(p):
> ```

**Review by michalbiesek — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/memkind/memkind/611)* status: 8 of 13 files reviewed, 6 unresolved discussions (waiting on @bratpiorka, @jacekgrzybek, and @michalbiesek)

**Pull request merged**